docs/typelibrary/utils/math/OFFSET.md

```
@@ -0,0 +1,52 @@
+# OFFSET
+
+![OFFSET](https://github.com/user-attachments/assets/de1b357e-6c34-406d-a094-bdf42c333011)
+
+* * * * * * * * * *
+## Einleitung
+Der OFFSET-Funktionsblock ist ein einfacher Baustein zur Berechnung der Differenz zwischen einem Eingangswert und einem gespeicherten Erstwert. Er dient hauptsächlich zur Offset-Korrektur von Signalen.
+
+## Schnittstellenstruktur
+
+### **Ereignis-Eingänge**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)